cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
This is a good way to validate the data for the `Ticket` model. It ensures the ticket's row, seat, and cinema hall are valid before saving the ticket into the database. This helps to maintain the data integrity of the application.